packages/lodestar-cli/src/db.ts

```diff
@@ -0,0 +1,34 @@
+import {fromHexString} from "@chainsafe/ssz";
+import {BeaconDb, LevelDbController} from "@chainsafe/lodestar/lib/db";
+import {WinstonLogger, LogLevel} from "@chainsafe/lodestar-utils";
+
+import {getBeaconPaths} from "./cmds/beacon/paths";
+import {mergeConfigOptions} from "./cmds/beacon/config";
+import {getBeaconConfig} from "./util";
+
+const rootDir = ".altona";
+const bucket = "badBlock";
+const action = "delete";
+const value = "0x3edb8564a3f5b9ea67f80477b631b72a0d602dcf6b95ab6a8c0666c273b1f876";
```

Comment:
maybe we should find a way to pass this as a param later

Reply:
whoops, I accidentally added these files, they're just some ideas I was playing with locally. Will remove them.